8-lambda/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM amazonlinux:2017.03.1.20170812
+
+ENV NODE_VERSION 8.10.0
+ENV SERVICE_ROOT /service
+ENV SERVICE_USER service
+ENV YARN_VERSION 1.6.0
+
+RUN curl --silent --location https://rpm.nodesource.com/setup_8.x | bash -
```

Comment:
@articulate/platform any concerns with installing node this way?

Reply:
i would be more inclined to leave the install version locked to a major release instead of using latest.

Reply:
Yeah, both `node` and `amazonlinux` are locked to specific versions.  @bellaweo, I think @pklingem was asking more about security concerns with this installation method, etc.  That's a question I had, too.

Reply:
I have no concerns with curl bash so long as it is over https